docs/variables.md

```diff
@@ -39,6 +39,7 @@ This page documents the variable substitutions supported by `Tasks` and `Pipelin
 | `context.taskRun.namespace` | The namespace of the `TaskRun` that this `Task` is running in. |
 | `context.taskRun.uid` | The uid of the `TaskRun` that this `Task` is running in. |
 | `context.task.name` | The name of this `Task`. |
+| `tasks.<pipelineTaskName>.status` | The execution status of the specified `pipelineTask`, only available in `finally` tasks. |
```

Comment:
Shouldn't this be under "Variables available in a `Pipeline`" ?

Reply:
I don't know, I was thinking a task here means a pipelineTask and that's why added it here.

Reply:
The two sections describe variables available when authoring pipelines and tasks respectively. 
All the examples (including in the TEP) show that the variable is used within a Pipeline, which is correct because a task cannot know about `pipelineTask` names, unless perhaps it's an embedded one. 
So this variable should be moved to the pipeline section.

Reply:
thanks @afrittoli I have moved this to pipeline section 🙏